packages/translate/src/index.js

```diff
@@ -164,12 +164,14 @@ Translate.prototype.detect = function(input, callback) {
 /**
  * Get an array of all supported languages.
  *
- * @resource [Discover Supported Languages]{@link https://cloud.google.com/translate/v2/using_rest#supported-languages}
+ * @resource [Discovering Supported Languages]{@link https://cloud.google.com/translate/v2/discovering-supported-languages-with-rest}
  *
+ * @param {string=} target - Include the languages names in the response
```

Comment:
I think this one is okay. I'll post a warning on Twitter just in case anyone is listening :)